series.tex

```diff
@@ -51,6 +51,7 @@ \subsubsection{Series Expansion}
 \texttt{sympy.polys.rs\_series} takes as input any SymPy expression and hence
 there is no need to explicitly create a polynomial \texttt{ring}. An example:
 
+%no-doctest
```

Comment:
I removed this and got

```
File "/Users/aaronmeurer/Documents/Python/sympy/sympy-paper/test_full_paper.py", line 291, in test_full_paper
Failed example:
    rs_series(sin(a + b), a, 4)
Expected:
    -1/2*(sin(b))*a**2 + (sin(b)) - 1/6*(cos(b))*a**3 + (cos(b))*a
Got:
    -1/6*(cos(b))*a**3 + (cos(b))*a - 1/2*(sin(b))*a**2 + (sin(b))
```

I guess there's a bug here. Sometimes it shows one output and sometimes it shows the other. Did you open an issue for this?